packages/core/src/dialects/abstract/query-interface.d.ts

```diff
@@ -627,4 +631,10 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
    * Lists all available databases
    */
   listDatabases(options?: QueryRawOptions): Promise<DatabaseDescription[]>;
+
+  assertTableHasColumn(
```

Comment:
Based on how our other methods are named, this should just be named `describeColumn`

That method already exists though, so a simple TODO is enough for this PR

Reply:
Done